app/views/pages/faq.html.erb

```diff
@@ -418,7 +418,7 @@
       </p>
       <p>
         So, we created a platform (this website), linking Kitt and Advent of Code, to build
-        <%= link_to "custom agregated scoreboards", scores_path, class: "link-explicit link-internal" %><sup>(*)</sup>.
+        <%= link_to "custom agregated scoreboards", "#", class: "link-explicit link-internal" %><sup>(*)</sup>.
```

Comment:
Typo
```suggestion
        <%= link_to "custom aggregated scoreboards", "#", class: "link-explicit link-internal" %><sup>(*)</sup>.
```

Reply:
Seems like I reworded later on, it's not on `main` anymore :D